CHANGELOG.md

```diff
@@ -25,6 +25,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.28.8] - 2024-07-03
```

Comment:
I took this version when I merged #228 to main, so you'll have to rebase again, sorry.

If you want, you could merge all the currently open dependabot PRs on this repo into a single PR and create just one new version in the changelog mentioning all the changes.

Reply:
No worries. Will do rebase
will be merging jinja2 as a separate one as I already used dependabot branch itself and all others in one PR